src/modules/ROOT/pages/changelog/202303/08.json

```
@@ -0,0 +1,66 @@
+{
+  "meta": {
+    "releasedOn": "2023.10.18"
+  },
+
+  "changed": [
+    {
+      "what": [
+        "record import endpoints by replacing the old endpoints with new ones."
+      ],
+      "why": [
+        "to optimize the process of importing records by making it more stable, efficient and capable of handling large amounts of record imports."
```

I was thought that you put a comma before the `and` if there are more then 2 items; so `a, b and c` becomes `a, b, and c`

```suggestion
        "to optimize the process of importing records by making it more stable, efficient, and capable of handling large amounts of record imports."
```